backend-api/src/main/resources/application.yml

```diff
@@ -0,0 +1,12 @@
+server:
+  port: 8080
+
+cassandra:
+  contactpoints: localhost
+  port: 9042
+  keyspace: pet_mgt
+
+toggles:
+  - name: activate-cassandra-rw
+    active: false
+
```

Comment:
I think we should try using typesafe config instead (https://github.com/lightbend/config) which is used in a lot of libraries (like datastax java driver)

Reply:
I agree I didn't that library. This is just a draft, for now I am just playing with configs. But I will change it, it makes more sense in the end.

Reply:
We will also have fewer annotations spread around the config code which is a great plus for this.

Reply:
@joao-r-reis Can you check?

It is kinda ugly but I will find a better approach to ConfigLoading instead of reading per file the config.